Makefile

```diff
@@ -20,3 +31,46 @@ test:
 install:
 	go install -v ./pkg/... ./cmd/...
 
+.PHONY: clean
+clean:
+	rm -rf *.deb *.rpm *.tar.gz ./release
+
+.PHONY: build
+build: clean ## Create release artefacts for darwin:amd64, linux:amd64 and windows:amd64. Requires etcd, glide, hg.
+	go run ./cmd/apiserver-builder-release/main.go vendor --version $(VERSION)
```

Comment:
The preferred format of this command is to pass it a `--commit` flag to make sure it pulls down the apiserver-builder at the intended versions (and not whatever glide has cached)

Reply:
Can you explain how we could make use of `--commit` here? Which version would be expected?

Reply:
TL;DR
The most recent commit hash from the apiserver-buidler repo

The release command doesn't use the local src for building the glide.tar.gz with the vendored go code for apiserver-builder++.  Instead it uses apiserver-boot (built from the local code) to start a new project, and run `glide` to download all of the godeps to vendor.  The commit here is put in the glide.yaml file to pin the apiserver-builder vendored go libraries.  This ensures that the correct version of the libraries are vendored, instead of whatever glide decides it wants to fetch (can depend on the local cache).